src/Features/CSharp/Portable/UseAutoProperty/CSharpUseAutoPropertyAnalyzer.cs

```diff
@@ -66,15 +66,13 @@ protected override bool CanExplicitInterfaceImplementationsBeFixed()
             List<AnalysisResult> analysisResults, HashSet<IFieldSymbol> ineligibleFields,
             Compilation compilation, CancellationToken cancellationToken)
         {
-            var groups = analysisResults.Select(r => (TypeDeclarationSyntax)r.PropertyDeclaration.Parent)
+            var groups = analysisResults.Select(r => (typeDeclaration: (TypeDeclarationSyntax)r.PropertyDeclaration.Parent, r.SemanticModel))
```

Comment:
Just curious: how does this relate to updating analyzers? Was this flagged by some new analyzer?